templates/repo/commit_page.tmpl

```diff
@@ -66,14 +66,15 @@
 										</div>
 										<div class="content">
 											<p id="cherry-pick-content" class="branch-dropdown"></p>
-											{{template "repo/branch_dropdown" dict "root" .
-												"noTag" true "disableCreateBranch" true
-												"branchForm" "branch-dropdown-form"
-												"branchURLPrefix" (printf "%s/_cherrypick/%s/" $.RepoLink .CommitID) "branchURLSuffix" ""
-												"setAction" true "submitForm" true}}
-											<form method="get" action="{{$.RepoLink}}/_cherrypick/{{.CommitID}}/{{if $.BranchName}}{{PathEscapeSegments $.BranchName}}{{else}}{{PathEscapeSegments $.Repository.DefaultBranch}}{{end}}" id="branch-dropdown-form">
-												<input type="hidden" name="ref" value="{{if $.BranchName}}{{$.BranchName}}{{else}}{{$.Repository.DefaultBranch}}{{end}}">
-												<input type="hidden" name="refType" value="branch">
+
+											<form method="get">
+												{{template "repo/branch_dropdown" dict
+													"Repository" .Repository
+													"ShowTabBranches" true
+													"CurrentRefType" "branch"
```

Comment:
looks like start showing from branch tab is not correct. when i want to see source from tag, dropdown button still shows branch name

Reply:
Nope, this selector is use to choose the base branch to receive a cherry-pick commit. It is not used for "source of tags".

This is the same behavior as old code, old code used `"noTag" true` to disable the tag tab and only show branch tab.